### PR TITLE
[CAM-12582] Tasklist in Hindi Language

### PR DESCRIPTION
**Pull request by anchal-gupta**
Please review tasklist in hindi language and if it works for you, please merge it. If it does not work for you, please let me know discrepancies you find. I will be happy to improve :)

### CODE REVIEW

**Comment by tmetzke** [comment]
Hey @anchal-gupta,

thanks for that awesome contribution, we appreciate it!

We will have a closer look and provide proper feedback as soon as we can. :+1:

Thanks and best,
Tobias

File: tasklist/hi.json [hunk 1]
```diff
@@ -0,0 +1,520 @@
+{
+  " लेबल " : {
```
Comment (marstamm) [inline tasklist/hi.json:2]: The 'labels' property is not a translate string. The translations will not be found by the Application if the top-level attributes are changed

```suggestion
  "labels" : {
```

File: tasklist/hi.json [hunk 2]
```diff
@@ -0,0 +1,520 @@
...
+    " APP_VENDOR " : " ऐप विक्रेता " ,
```
Comment (marstamm) [inline tasklist/hi.json:3]: The translation label should be without leading or trailing spaces. I can't speak for the translation, please make sure that this is correct as well.

```suggestion
    "APP_VENDOR" : " ऐप विक्रेता " ,
```

This applies to all labels

File: tasklist/hi.json [hunk 3]
```diff
@@ -0,0 +1,520 @@
...
+    " PAGES_MSG_REQUEST_TIMEOUT " : " । आपका अनुरोध का समय समाप्त पृष्ठ को ताज़ा करने की कोशिश करें। " ,
+    " PAGES_STATUS_ACCESS_DENIED " : " प्रवेश निषेध " ,
+    " PAGES_MSG_ACCESS_DENIED " : " आप {{missingAuths}} में अनधिकृत कर रहे हैं। " ,
+    " PAGES_MSG_ACCESS_DENIED_RESOURCE_ID " : " आप {{permissionName}} {{ResourceName}} {{RESOURCEID}} में अनधिकृत कर रहे हैं। " ,
+    " PAGES_MSG_ACTION_DENIED " : "  एक क्रिया का कार्यान्वयन सर्वर द्वारा नहीं दी गई है पृष्ठ को ताज़ा करने का प्रयास करें " ,
+    " PAGES_STATUS_NOT_FOUND " : " नहीं मिला " ,
+    " PAGES_MSG_NOT_FOUND " : "  एक आपके द्वारा अनुरोधित संसाधन नहीं पाया जा सका " ,
+    " PAGES_STATUS_COMMUNICATION_ERROR " : " संचार त्रुटि " ,
+    " PAGES_MSG_COMMUNICATION_ERROR " : " इस अनुप्रयोग में अनपेक्षित {{स्थिति}} सर्वर से प्रतिक्रिया प्राप्त पेज या लॉगिन और आवेदन के बाहर ताज़ा करने के लिए प्रयास करें " ,
+    " PAGES_MSG_ENGINE_NOT_EXISTS " : " जिस प्रक्रिया इंजन को आप एक्सेस करने का प्रयास कर रहे हैं वह मौजूद नहीं है , " ।
```
Comment (marstamm) [inline tasklist/hi.json:72]: The delimiter is wrong and the file no longer a valid JSON

```suggestion
    " PAGES_MSG_ENGINE_NOT_EXISTS " : " जिस प्रक्रिया इंजन को आप एक्सेस करने का प्रयास कर रहे हैं वह मौजूद नहीं है , ",
```

File: tasklist/hi.json [hunk 4]
```diff
@@ -0,0 +1,520 @@
...
+    " CAM_WIDGET_VARIABLES_TABLE_EDIT_VARIABLE " : " चर को संपादित करें " ,
+    " CAM_WIDGET_VARIABLES_TABLE_DELETE_VARIABLE " : " चर हटाएं " ,
+    " CAM_WIDGET_VARIABLES_TABLE_SAVE_VARIABLE " : " चर सहेजें " ,
+    " CAM_WIDGET_VARIABLES_TABLE_REVERT_VARIABLE " : " चर को वापस लाएं " ,
+    " CAM_WIDGET_VARIABLES_TABLE_NAME " : " नाम " ,
+    " CAM_WIDGET_VARIABLES_TABLE_TYPE " : " टाइप " ,
+    " CAM_WIDGET_VARIABLES_TABLE_VALUE " : " मान " ,
+    " CAM_WIDGET_VARIABLES_TABLE_DELETE " : " हटाएं " ,
+    " CAM_WIDGET_VARIABLES_TABLE_ABORT " : " बीच में बंद करें " ,
+    " CAM_WIDGET_VARIABLES_TABLE_DIALOGUE " : " यकीन है कि आप '{{नाम}}' को हटाना चाहते हैं कर रहे हैं " ,
```
Comment (marstamm) [inline tasklist/hi.json:113]: Placeholders are not translated, as the replacement will no longer work

File: tasklist/hi.json [hunk 5]
```diff
@@ -0,0 +1,520 @@
...
+    " CAM_WIDGET_VARIABLE_DIALOG_INSPECT " : " {{name}} चर " का निरीक्षण करें ,
+    " CAM_WIDGET_VARIABLE_DIALOG_LABEL_OBJECT_TYPE " : " ऑब्जेक्ट प्रकार का नाम: " ,
+    " CAM_WIDGET_VARIABLE_DIALOG_LABEL_SERIALIZATION " : " सीरियल डेटा स्वरूप: " ,
+    " CAM_WIDGET_VARIABLE_DIALOG_LABEL_SERIALIZED_VALUE " : " धारावाहिक मूल्य " ,
+    " CAM_WIDGET_VARIABLE_DIALOG_BTN_CLOSE " : " बंद करें , "
+    " CAM_WIDGET_VARIABLE_DIALOG_BTN_CHANGE " : " बदलें " ,
+    " CAM_WIDGET_VARIABLE_NULL " : " <null> " ,
```
Comment (marstamm) [inline tasklist/hi.json:120]: Please trim the labels
```suggestion
    "CAM_WIDGET_VARIABLE_NULL" : "<null>" ,
```

File: tasklist/hi.json [hunk 6]
```diff
@@ -0,0 +1,520 @@
...
+    " CAM_WIDGET_VARIABLE_SET_VALUE_NULL " : " करने के लिए सेट मूल्य \" अशक्त \ " " ,
+    " CAM_WIDGET_VARIABLE_VALUE " : " चर का मान " ,
+    " CAM_WIDGET_VARIABLE_UNDEFINED " : " <अपरिभाषित> , "
+    " CAM_WIDGET_VARIABLE_RESET " : " पिछले या डिफ़ॉल्ट मान पर पुनः सेट करें , "
+    " CAM_WIDGET_STRING_DIALOG_TITLE " : " निरीक्षण \" {{नाम}} \ " चर " ,
```
Comment (marstamm) [inline tasklist/hi.json:125]: Translated Placeholder

File: tasklist/hi.json (hunk 4 above)
Comment (marstamm) [reply tasklist/hi.json:113]:
```suggestion
    "CAM_WIDGET_VARIABLES_TABLE_DIALOGUE" : " यकीन है कि आप '{{name}}' को हटाना चाहते हैं कर रहे हैं " ,
```

File: tasklist/hi.json [hunk 7]
```diff
@@ -0,0 +1,520 @@
...
+    " TELEMETRY_PREFERENCES_META " : " मेटा और उपयोग सांख्यिकी सक्षम करें " ,
+    " TELEMETRY_PREFERENCES_META_DETAILS " : " Camunda बीपीएम रनटाइम उपयोग के आंकड़े भेजने की अनुमति दें हम इस जानकारी का उपयोग विशेष रूप से आप पर्यावरण प्रयोग कर रहे हैं में एक स्थिर और बेहतर उत्पाद अनुभव प्रदान करने के लिए इस उत्पाद संस्करण के बारे में कलेक्ट जानकारी, तकनीकी वातावरण और कैसे Camunda अनुमति देता है।। सुविधाओं उपयोग में हैं। यह जानकारी आम तौर पर तकनीकी और एकीकृत की है, लेकिन सीमित है और व्यक्तिगत डेटा pseudonymized शामिल हो सकते हैं। " ,
+    " TELEMETRY_PREFERENCES_DISCLAIMER " : " इन विकल्पों में, आपकी व्यक्तिगत जानकारी में से कोई भी प्रस्तुत किया जाएगा के किसी भी है। " ,
+    " TELEMETRY_FOOTER_LEARN_MORE " : "और जानें: " ,
+    " TELEMETRY_FOOTER_PRIVACY_POLICY " : " कैमुंडा गोपनीयता नीति " ,
+    " TELEMETRY_ERROR_STATUS " : " त्रुटि " ,
+    " TELEMETRY_ERROR_MESSAGE " : " आपका कॉन्फ़िगरेशन सहेजा नहीं जा सका। कृपया पुनः प्रयास करें। "
+  },
+  " डेटक्लोलेस " : {
+    " महीने " : [
```
Comment (marstamm) [inline tasklist/hi.json:430]: Same as with the 'labels' property, don't translate the property names

```suggestion
  "dateLocales": {
    "months": [
```

File: welcome/hi.json [hunk 8]
```diff
@@ -0,0 +1,164 @@
...
+    " DESC " : " अवरोही " ,
+    " BOOLEAN " : " बुलियन " ,
+    " SHORT " : " लघु " ,
+    " INTEGER " : " इंटेगर " ,
+    " LONG " : " लंबी " ,
+    " DOUBLE " : " डबल " ,
+    " DATE " : " दिनांक " ,
+    " STRING " : " वर्ण " ,
+    " AUTH_LOGOUT_SUCCESSFUL " : " समापन सत्र सफल " ,
+    " AUTH_LOGOUT_THANKS " : " आज कैमुंडा का उपयोग करने के लिए धन्यवाद। एक महान {{dayConxtxt}}! " ,।
```
Comment (marstamm) [inline welcome/hi.json:39]: Broken JSON
```suggestion
    " AUTH_LOGOUT_THANKS " : " आज कैमुंडा का उपयोग करने के लिए धन्यवाद। एक महान {{dayConxtxt}}! " ,
```

File: welcome/hi.json [hunk 9]
```diff
@@ -0,0 +1,164 @@
+{
+  " लेबल " : {
```
Comment (marstamm) [inline welcome/hi.json:2]: Translated Property

File: welcome/hi.json [hunk 10]
```diff
@@ -0,0 +1,164 @@
...
+    " CAM_WIDGET_VARIABLES_TABLE_DELETE_VARIABLE " : " चर हटाएं " ,
+    " CAM_WIDGET_VARIABLES_TABLE_SAVE_VARIABLE " : " चर सहेजें " ,
+    " CAM_WIDGET_VARIABLES_TABLE_REVERT_VARIABLE " : " चर को वापस लाएं " ,
+    " CAM_WIDGET_VARIABLES_TABLE_NAME " : " नाम " ,
+    " CAM_WIDGET_VARIABLES_TABLE_TYPE " : " टाइप " ,
+    " CAM_WIDGET_VARIABLES_TABLE_VALUE " : " मान " ,
+    " CAM_WIDGET_VARIABLES_TABLE_DELETE " : " हटाएं " ,
+    " CAM_WIDGET_VARIABLES_TABLE_ABORT " : " बीच में बंद करें " ,
+    " CAM_WIDGET_VARIABLES_TABLE_DIALOGUE " : " क्या आप वाकई हटाना चाहते हैं '{{नाम}}' ' ,
+    " CAM_WIDGET_VARIABLE_DIALOG_INSPECT " : " का निरीक्षण चर {{नाम}} " ,
```
Comment (marstamm) [inline welcome/hi.json:114]: Translated Placeholders

**Review by marstamm — CHANGES_REQUESTED** [review]
Hi @anchal-gupta, thank you for you contribution! We appreciate the effort that went into this.

However, there are some technicalities which will prevent the translation from being used. Please make sure that the [translation file works in Tasklist](https://docs.camunda.org/manual/7.14/webapps/tasklist/configuration/#localization) when workin on it.
I added some hints below.

Cheers
Martin

**Comment by marstamm** [comment]
Hi @anchal-gupta ,

as 2 weeks have passed without updates to the PR, I am going to close it. Feel free to reopen it when you integrated the review hints.

Cheers
Martin

**Pull request closed without merging**